opentelemetry-sdk/tests/logs/test_log_record.py

```diff
@@ -31,7 +31,7 @@ def test_log_record_to_json(self):
                 "trace_id": "",
                 "span_id": "",
                 "trace_flags": None,
-                "resource": "",
+                "resource": None,
```

Comment:
Might be nice to add a test to verify that a non-empty Resource also converts to json correctly.

Reply:
Yep, certainly agree that coverage was missing. As a first time contributor here I was trying to things slim, but I'm happy to add it:

https://github.com/open-telemetry/opentelemetry-python/pull/3346/files#diff-c202e3816c113727d0d32d83fa4becda1632ae14c6bf62b7421e96cb936b2c85R45